plugins/mock/src/mockDatabase.ts

```diff
@@ -0,0 +1,42 @@
+import type {
+    BasePluginArgs,
+    Bundle,
+    DatabasePlugin,
+    DatabasePluginHooks
+  } from '@hot-updater/plugin-core';
+  
+  export interface MockDatabaseConfig {
+    initialBundles?: Bundle[];
+  }
+  
+  export const mockDatabase =
+  (config: MockDatabaseConfig, hooks?: DatabasePluginHooks) => (_: BasePluginArgs): DatabasePlugin => {
+    let bundles: Bundle[] = config.initialBundles ?? [];
```

Comment:
Shouldn’t there be two bundle lists stored in memory? There should be a confirmed bundle and a changeset bundle list, so that during commitBundles, the changeset bundle replaces the confirmed bundle.